image_proc/launch/image_proc.launch.py

```diff
@@ -30,36 +30,89 @@
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-from launch import LaunchDescription
+import os
+import sys
+
+import launch
+from launch import LaunchDescription, LaunchIntrospector, LaunchService
+from launch_ros import actions, get_default_launch_description
+
 from launch_ros.actions import ComposableNodeContainer
 from launch_ros.descriptions import ComposableNode
 
+# Set namespace for all nodes
+NAMESPACE = "/camera"
 
 def generate_launch_description():
-    return LaunchDescription([
-        ComposableNodeContainer(
-            package='rclcpp_components', node_executable='component_container',
-            node_name='image_proc_container', node_namespace='',
-            composable_node_descriptions=[
-                ComposableNode(
-                    package='image_proc',
-                    node_plugin='image_proc::DebayerNode',
-                    node_name='debayer'
-                ),
-                ComposableNode(
-                    package='image_proc',
-                    node_plugin='image_proc::RectifyNode',
-                    node_name='rectify_mono'
-                ),
-                ComposableNode(
-                    package='image_proc',
-                    node_plugin='image_proc::RectifyNode',
-                    node_name='rectify_color',
-                    remappings=[
-                        ('image_mono', 'image_color'),
-                        ('image_rect', 'image_rect_color')
-                    ]
-                ),
-            ],
-        ),
-    ])
+    """
+    Generate a launch description for image processing nodes.
+    """
+    ld = LaunchDescription()
+
+    # Load composable container
+    image_processing = actions.ComposableNodeContainer(
```

Comment:
I'm not trying to be difficult, but I don't understand. You remove the parameter, OK, but I don't see any remaps here to actually enable things to change those default values. 

It seems to be that what you want is no parameters (like in the original launch file here) and use remappings of the topics. It seems like the use of `camera_namespace` is strange. 

How do your changes make this work where the original did not?

Reply:
I get what you're saying, I suppose I'm not sure of the value of this change, though I think an example with a remap would be valuable, which I'll be changing this too. What made my code work (on another project) was the remapping changes, which I didn't implement in this launch file. 

The `camera_namespace` keeps things a little more readable for me at least, and when used in remaps, prevents repetitive use of the same string for the namespace. I'll push an example tonight.

All in all, I don't mind the criticism, this is my first open source PR, so I'm still learning the ropes :), thanks for being accommodating!

Reply:
@caelinsutch You can always make a launch file that has the namespace your pre-pend to the new remap target if you like (e.g. `{image, NAMESPACE + /remapped_name}`). But for the general case doing something like `{image, front_cam/image}` makes alot more sense to have remap abilities rather than independent namespace and image topic parameters. 

I think the right answer is to delete all the namespace parameters and then replace them with remaps like in the original launch file here. I think we added the namespaces here during a port before remap was available.

Reply:
So would you like me to do that in this PR, or make a new where I delete namespace params in the image_proc node

Reply:
We should do it in this PR, I'm not sure this works as is without it. 

I think what should be done:
- Remove namespace parameter (then all pub/subs should be just "image" and "camera_info")
- Launch file has remaps for "image" and "camera_info" as needed for namespacing or changing to `_mono` `_color` for use in the pipeline

Reply:
Sounds good, I'll likely write that up and write a PR for it tomorrow :)

Reply:
Updated with removed namespace for image_proc and added launch file remaps